Datastore/src/V1/resources/datastore_rest_client_config.php

```diff
@@ -88,51 +88,5 @@
                 ],
             ],
         ],
-        'google.longrunning.Operations' => [
```

Comment:
These configs here were probably removed erroneously, and we need to update the bazel target to include the service JSON to get them back.